test/token/ERC1155/ERC1155Enumerable.behavior.js

```diff
@@ -20,22 +24,36 @@ const describeBehaviorOfERC20Metadata = function ({ deploy }, skips) {
     }, skips);
 
     describe('#totalSupply', function () {
-      it('returns supply of given token');
+      it('returns supply of given token', async function(){
+        let id = ethers.constants.Zero;
+        expect(await instance.callStatic.totalSupply(id)).to.equal(2);
```

Comment:
Don't hardcode the value of `2` if it's defined in a constant above (`amount`, line 17).